src/twisted/persisted/test/test_styles.py

```diff
@@ -125,4 +122,4 @@ def test_lambdaRaisesPicklingError(self):
         """
         Pickling a C{lambda} function ought to raise a L{pickle.PicklingError}.
         """
-        self.assertRaises(pickle.PicklingError, pickle.dumps, lambdaExample)
+        self.assertRaises(pickle.PicklingError, pickle.dumps, lambda x: x)
```

Comment:
so this is a bit odd - see https://freenode.logbot.info/twisted-dev/20201026#c5600809-c5603273